src/Core/src/Sts/Input/AssumeRoleWithWebIdentityRequest.php

```diff
@@ -240,11 +240,18 @@ public function setWebIdentityToken(?string $value): self
 
     public function validate(): void
     {
-        foreach (['RoleArn', 'RoleSessionName', 'WebIdentityToken'] as $name) {
-            if (null === $this->$name) {
```

Comment:
Did we remove “name”?

Reply:
`"name"`?.. you mean `$name` which is a variable containing either `RoleArn`, `RoleSessionName` or `WebIdentityToken`. The check is still performed bellow